tests/functional/serving/test_root_resource.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""Perform functional tests for the REST endpoint `/`"""
+
+
+def test_orion_version(client):
```

Comment:
Do you think merging these 3 tests in one would be simpler and more efficient?

Reply:
Sure :). I was doing TDD so it was handy to have one of each, especially since there is no variations to test but one test also works 👍